app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+import datetime
```

Comment:
It is best practice to write built-in modules first

Reply:
Fix imports order